src/test/unit/bisect.test.js

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import { bisectionLeft, bisectionRight } from '../../utils/bisect';
+
+describe('bisectionRight', function() {
+  const array = [0, 1, 2, 3, 3, 3, 3, 3, 5, 6, 7, 8, 9];
+  // The new element to be inserted into this array is referred to as x
+  it('returns index of the first number greater than x', function() {
+    expect(bisectionRight(array, 4)).toBe(8);
+    expect(bisectionRight(array, 3)).toBe(8);
+  });
+
+  it('returns index of the first number greater than x, occuring after low', function() {
+    expect(bisectionRight(array, 4, 8)).toBe(8);
+    expect(bisectionRight(array, 3, 9)).toBe(9);
+    expect(bisectionLeft(array, 2, -100)).toBe(2);
+  });
+
+  it('returns index of the first number greater than x, between low and high', function() {
+    expect(bisectionRight(array, 4, 1, 7)).toBe(7);
+    expect(bisectionRight(array, 3, 4, 6)).toBe(6);
+  });
+
+  it('returns 0 if all elements are greater than x', function() {
+    expect(bisectionLeft(array, -5)).toBe(0);
+  });
```

Comment:
Could you please also add a test with a `x` that's bigger than all other numbers?
(same for bisectionRight).
It should return `array.length` in both cases.